src/hotspot/share/gc/g1/g1BarrierSet.cpp

```diff
@@ -120,6 +120,12 @@ void G1BarrierSet::invalidate(MemRegion mr) {
   volatile jbyte* byte = _card_table->byte_for(mr.start());
   jbyte* last_byte = _card_table->byte_for(mr.last());
   Thread* thr = Thread::current();
+  if (G1BarrierSimple) {
+    for (; byte <= last_byte; byte++) {
+      *byte = G1CardTable::dirty_card_val();
```

Comment:
a memset could be used: memset((void*)byte, G1CardTable::dirty_card_val(), last_byte - byte + 1);

Reply:
fixed